sr/robot3/serial_wrapper.py

```diff
@@ -106,6 +107,8 @@ def __init__(
             do_not_open=True,
         )
 
+        atexit.register(self._disconnect, quiet=True)
```

Comment:
Thought: If this handler is called before the board's `_cleanup`, it'll act on a closed socket. Should calling `_disconnect` be handled by the board instead?

Reply:
Board cleanups are registered after the socket is created so the disconnect will be called after the cleanup.